subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -775,6 +783,13 @@ static int bt_mesh_lightness_srv_start(struct bt_mesh_model *mod)
 	       set.lvl, transition.time);
 
 	lightness_srv_change_lvl(srv, NULL, &set, &dummy);
+
+	BT_DBG("Publishing Light %s to 0x%04x", repr_str[ACTUAL],
+	       srv->pub.addr);
+
+	/* Publishing is always done as an Actual, according to test spec. */
+	pub(srv, NULL, &dummy, ACTUAL);
+
```

Comment:
This must also be added to the gen_lvl callbacks

Reply:
Okay. I've also added publish to `lvl_move_set` callback from light_temp_srv.c.